api/models.py

```diff
@@ -23,7 +23,7 @@ class Loan_Record(models.Model):
     due_date=models.DateField()
     created=models.DateField(auto_now_add=True)
     amount=models.IntegerField()
-    interest_rate=models.DecimalField(max_digits=5, decimal_places=2,)
+    interest_rate=models.CharField(max_length=10 , blank=True,null=True) 
```

Comment:
It was change from DecimalField to CharField because of this error **invalid literal for int() with base 10: '11.00'**